### PR TITLE
Update UI of comment page. Add pictures to comments

### PR DESCRIPTION
**Pull request by swalsh15**
Styled comment page to look more like UI of actual comment page. Updated servlet to send user picture to frontend.

Screenshot 2020-06-04 at 11 14 59 AM - Display 2
Screenshot 2020-06-04 at 11 15 08 AM - Display 2

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -44,10 +44,12 @@
   private final class Comment {
```
Comment (g-harel) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:44]: Just saw this, very nice!

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -54,15 +54,24 @@ function fetchComments() {
...
+      userInfo.setAttribute("class", "user-info");
+
+      const picture = document.createElement("img");
+      picture.setAttribute("class", "comment-picture");
+      picture.src = comment.pictureURL;
+
       const user = document.createElement("div");
       user.setAttribute("class", "user");
-      user.textContent = comment.user + " said:";
+      user.textContent = comment.user;
```
Comment (g-harel) [inline portfolio/src/main/webapp/script.js:66]: I think `createTextNode` would be a better match, textContent is not really meant for this.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -94,11 +94,28 @@ main {
 }
 
 .user {
-  border-bottom: dashed;
-  border-width: 2px;
-  padding: 2% 0%;
+  align-self: center;
+  font-size: 120%;
```
Comment (g-harel) [inline portfolio/src/main/webapp/style.css:98]: Nit: Using `em` would be more common for fonts.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (Danish21) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:44]: +1  looks great!

**Pull request merged**